ocs_ci/ocs/cluster.py

```diff
@@ -954,6 +954,34 @@ def validate_replica_data(pool_name, replica):
     raise PoolNotFound(f"Pool {pool_name} not found on cluster")
 
 
+def calculate_compression_ratio(pool_name):
+    """
+    Calculating the  compression of data on RBD pool
+
+    Args:
+        pool_name (str): the name of the pool to calculate the ratio on
+
+    Returns:
+        int : the compression ratio n in percentage
+
+    """
+    results = get_ceph_df_detail()
+    for pool in results["pools"]:
+        if pool["name"] == pool_name:
+            used = pool["stats"]["bytes_used"]
+            used_cmp = pool["stats"]["compress_bytes_used"]
+            stored = pool["stats"]["stored"]
+            ratio = int((used_cmp * 100) / (used_cmp + used))
+            logger.info(f"pool name is {pool_name}")
+            logger.info(f"net stored data is {stored}")
+            logger.info(f"total used data is {used}")
+            logger.info(f"compressed data is {used_cmp}")
+            return ratio
+
+    logger.warning(f"the pool {pool_name} does not exits !")
+    return 0
```

Comment:
I think it's better to return `None` in case the pool doesn't exist

Reply:
done